### PR TITLE
[Synthetics] fix mapping from long to float

### PR DESCRIPTION
**Pull request by shahzad31**
we made a mistake instead of using float we are using long for decimal values.

for PR https://github.com/elastic/integrations/pull/11878

### CODE REVIEW

File: packages/synthetics/data_stream/browser/fields/synthetics.yml [hunk 1]
```diff
@@ -81,29 +81,29 @@
...
           fields:
             - name: us
-              type: long
+              type: float
         - name: load
           type: group
           description: duration of Load end event
           fields:
             - name: duration
-              type: long
```
Comment (shahzad31) [inline packages/synthetics/data_stream/browser/fields/synthetics.yml:102]: this we reverted to original

File: packages/synthetics/data_stream/browser/fields/synthetics.yml [hunk 2]
```diff
@@ -81,29 +81,29 @@
...
+              type: integer
             - name: us
-              type: long
```
Comment (shahzad31) [inline packages/synthetics/data_stream/browser/fields/synthetics.yml:104]: same here

**Review by emilioalvap — APPROVED** [review]
LGTM

**Review by maryam-saeidi — APPROVED** [review]
Rubber stamp from my side, as Emilio reviewed this PR.

**Pull request merged**